index.html

```diff
@@ -215,8 +216,204 @@ <h3>WebSocket Re-use</h3>
+              <td>"readProperty"</td>
+              <td>A string which denotes that this message is requesting a <code>readproperty</code> operation.</td>
+            </tr>
+            <tr>
+              <td><code>name</code></td>
+              <td>string</td>
+              <td>Mandatory</td>
+              <td>The name of the <a>Property</a> to read.</td>
+            </tr>
+          </tbody>
+        </table>
+        <pre class="example" title="Read property operation (Consumer to Thing)">
+          {
+            "thingID": "https://mythingserver.com/things/mylamp1",
+            "messageID": "c370da58-69ae-4e83-bb5a-ac6cfb2fed54",
+            "messageType": "readProperty",
+            "name": "on",
+            "correlationID": "5afb752f-8be0-4a3c-8108-1327a6009cbd"
+          }
+        </pre>
+        <p>When a <a>Thing</a> receives a <code>readProperty</code> message from a <a>Consumer</a>, then upon
+          successfully reading the value of the corresponding property it MUST send a <a
+            href="propertyReading"><code>propertyReading</code></a> message in response, containing its current value.
+        </p>
+      </section>
+      <section id="propertyReading">
+        <h4><code>propertyReading</code></h4>
+        <p>To notify a <a>Consumer</a> of the value of a <a>Property</a>, a <a>Thing</a> MUST send a
+          <a href="#websocket-messages">message</a> to the <a>Consumer</a> which contains the following members:
+        </p>
+        <table class="def">
+          <caption>Members of a <code>propertyReading</code> message</caption>
+          <thead>
+            <tr>
+              <th>Member</th>
+              <th>Type</th>
+              <th>Assignment</th>
+              <th>Description</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td><code>messageType</code></td>
+              <td>string</td>
+              <td>"propertyReading"</td>
+              <td>A string which denotes that this message is notifying a <a>Consumer</a> of the value of a
+                <a>Property</a>.
+              </td>
+            </tr>
+            <tr>
+              <td><code>name</code></td>
+              <td>string</td>
+              <td>Mandatory</td>
+              <td>The name of the <a>Property</a> being read.</td>
+            </tr>
+            <tr>
+              <td><code>value</code></td>
+              <td>string</td>
+              <td>Mandatory</td>
+              <td>The current value of the <a>Property</a> being read, with a format conforming to the data schema of
+                the
+                corresponding PropertyAffordance in the <a>Thing Description</a>.</td>
+            </tr>
+            <tr>
+              <td><code>timestamp</code></td>
+              <td>string</td>
+              <td>Optional</td>
+              <td>A timestamp in date-time format [[RFC3339]] set to the time the property reading took place.</td>
```

Comment:
Do we requite UTC or any time zone can be used (allowed by this rfc)

Reply:
RFC 3339 only allows numerical offsets relative to UTC, and "Z" as timezones. I've added an assertion to the WebSocket Messages section to highlight this, consistent with WoT Profiles.